staging/src/k8s.io/apiextensions-apiserver/test/integration/subresources_test.go

```diff
@@ -443,58 +441,31 @@ func TestValidateOnlyStatus(t *testing.T) {
 		t.Fatalf("unable to create noxu instance: %v", err)
 	}
 
-	gottenCRD, err := apiExtensionClient.ApiextensionsV1beta1().CustomResourceDefinitions().Get("noxus.mygroup.example.com", metav1.GetOptions{})
+	// update the spec with .spec.num = 5, expecting no error
+	err = unstructured.SetNestedField(createdNoxuInstance.Object, int64(5), "spec", "num")
 	if err != nil {
-		t.Fatal(err)
-	}
-
-	// update the crd so that max value of spec.num = 5 and status.num = 10
-	gottenCRD.Spec.Validation.OpenAPIV3Schema = &apiextensionsv1beta1.JSONSchemaProps{
-		Properties: map[string]apiextensionsv1beta1.JSONSchemaProps{
-			"spec": {
-				Properties: map[string]apiextensionsv1beta1.JSONSchemaProps{
-					"num": {
-						Type:    "integer",
-						Maximum: float64Ptr(5),
-					},
-				},
-			},
-			"status": {
-				Properties: map[string]apiextensionsv1beta1.JSONSchemaProps{
-					"num": {
-						Type:    "integer",
-						Maximum: float64Ptr(10),
-					},
-				},
-			},
-		},
-	}
-
-	if _, err = apiExtensionClient.ApiextensionsV1beta1().CustomResourceDefinitions().Update(gottenCRD); err != nil {
```

Comment:
nit: I guess we should update the comment here then? https://github.com/sttts/kubernetes/blob/7c0bdfc2eaf8ca19d5526e32b8b6b2ead389782d/staging/src/k8s.io/apiextensions-apiserver/test/integration/subresources_test.go#L393 

This doesn't hold true anymore.

Reply:
fixed